### PR TITLE
 update packages

### PR DESCRIPTION
**Pull request by DFadeev**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 var gulp = require('gulp'),
-    livereload = require('gulp-livereload'),
+    clean = require('gulp-clean');
+livereload = require('gulp-livereload'),
```
Comment (Chernavskikh) [inline gulpfile.js]: @DFadeev please fix tab size

File: gulpfile.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
     connect = require('gulp-connect'),
     postcss = require('gulp-postcss'),
     rename = require('gulp-rename'),
-    cssnano = require('cssnano'),
-    postcssPresetEnv = require('postcss-preset-env'),
+    cleanCSS = require('gulp-clean-css');
```
Comment (Chernavskikh) [inline gulpfile.js]: @DFadeev replace ; with ,

File: package.json [hunk 3]
```diff
@@ -1,17 +1,18 @@
 {
   "name": "postcss-gulp-starter-kit",
-  "version": "2.1.0",
+  "version": "2.0.0",
```
Comment (Chernavskikh) [inline package.json]: please update version to 2.2.0

File: gulpfile.js (hunk 1 above)
Comment (DFadeev) [reply gulpfile.js]: @Chernavskikh fixed

File: gulpfile.js (hunk 2 above)
Comment (DFadeev) [reply gulpfile.js]: @Chernavskikh fixed

File: package.json (hunk 3 above)
Comment (DFadeev) [reply package.json]: @Chernavskikh updated

**Pull request merged**